### PR TITLE
Mombasa county budget data

### PR DESCRIPTION
**Pull request by ascii-dev**
Migrates Mombasa County Budget Data

### CODE REVIEW

File: runner.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SCRIPTS_DIR=./sql
+DATABASE_NAME=pesayetu
+
+for file in $SCRIPTS_DIR/*.sql
+    do sudo -u asciicfa psql $DATABASE_NAME -f $file
```
Comment (kilemensi) [inline runner.sh:7]: Why are we pushing files with _local_ configs?

**Review by kilemensi — APPROVED** [review]
:+1: @ascii-dev 

It's very difficult to review these data dumps like this... We can only review them once you've plotted viz from them in PesaYetu I think (applies to https://github.com/CodeForAfrica/HURUmap-Data/pull/12 as well)

QQ though:

1. Why do all files start with `py_`, and
2. Can we add newline to the end of CSV files or will that break our CSV parser.

File: runner.sh (hunk 1 above)
Comment (ascii-dev) [reply runner.sh:7]: Oh... This code was just something I used to run the scripts locally when we were testing the uppercase files and I was thinking of just keeping the code in the case we need to run all the `SQLs` sometime in the future... @kilemensi

**Comment by ascii-dev** [comment]
> 👍 @ascii-dev
> 
> It's very difficult to review these data dumps like this... We can only review them once you've plotted viz from them in PesaYetu I think (applies to #12 as well)
> 
> QQ though:
> 
> 1. Why do all files start with `py_`, and
> 2. Can we add newline to the end of CSV files or will that break our CSV parser.

We have `py_` in front to show that it's a `PesaYetu` data. @KhadijaMahanga said that's the convention she's been following. I'm not sure if a new line at the end of the `CSV` break the parser as I haven't even studied the parser, just using it for what it does...

**Pull request merged**